upgrades/schema/Version_8_0_20230323180900_add_column_is_deactivated_on_template_attribute.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pim\Upgrade\Schema;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * @copyright 2023 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class Version_8_0_20230323180900_add_column_is_deactivated_on_template_attribute extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->skipIf(
+            $schema->getTable('pim_catalog_category_attribute')->hasColumn('is_deactivated'),
+            'is_deactivated column already exists in pim_catalog_category_attribute'
+        );
+
+        $this->addSql('ALTER TABLE pim_catalog_category_attribute ADD COLUMN is_deactivated BOOLEAN;');
```

Comment:
I'm wondering if we are not supposed to set a DEFAULT value when adding a column?
```suggestion
        $this->addSql('ALTER TABLE pim_catalog_category_attribute ADD COLUMN is_deactivated BOOLEAN DEFAULT 0;');
```

Reply:
I made my migration based on the migration that adds the is_deactivated column on the template, I left the implicit default value which is null.

https://github.com/akeneo/pim-community-dev/blob/master/upgrades/schema/Version_8_0_20230131155436_add_column_is_deactivated_on_template.php